cg/services/order_validation_service/rules/case_sample/utils.py

```diff
@@ -59,6 +59,13 @@ def is_volume_invalid(sample: Sample) -> bool:
     return in_container and not allowed_volume
 
 
+def is_volume_missing(sample: Sample) -> bool:
```

Comment:
We could move this to cg/services/order_validation_service/rules/utils.py so as to remove some duplication

Reply:
Makes sense!